app/assets/javascripts/bastion/components/global-notification.service.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.components.service:GlobalNotification
+ *
+ * @description
+ *  Service to display a global notifcation
+ */
+angular.module('Bastion.components').service("GlobalNotification", [
+    function () {
+        this.errorMessages = [];
+        this.successMessages = [];
+
+        this.setSuccessMessage = function (message) {
+            this.successMessages.push(message);
```

Comment:
It's not consistent.  In some places we use `push()` and in others we replace the array with a new messages array.  You can end up with 5 success messages all stacked on top of each other on some pages in master and you must click the x on each one in order to dismiss it.  

I would like to see us fade out success messages after a certain amount of time or at the very least only show the latest one.